ggshield/core/git_shell.py

```diff
@@ -76,12 +76,23 @@ def git(
             [_get_git_path()] + command,
             check=check,
             stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
```

Comment:
I think we should leave this to `subprocess.PIPE` and use `e.stderr` to read the error message when catching the exception. This way if a normal, non failing, command prints to both stdout and stderr, (for example because it prints progress information to stderr), then the return of this function won't include the stderr output.